eva-server/src/test/java/uk/ac/ebi/eva/server/repository/VariantEntityRepositoryTest.java

```diff
@@ -63,7 +63,7 @@
  */
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration
-@UsingDataSet(locations = {"/test-data/variants.json"})
+@UsingDataSet(locations = {"/test-data/eva_hsapiens_grch37.variants_1_2.json"})
```

Comment:
I would prefer not to track the exact database structure in the code repository unless necessary. The collection may also be replaced or renamed in the future so it won't be of much help in the long term.